# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../acpitail/install.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ru acpitail-0.1.orig/Makefile acpitail-0.1/Makefile
--- acpitail-0.1.orig/Makefile	2007-06-18 12:07:30.000000000 +0200
+++ acpitail-0.1/Makefile	2007-08-06 20:37:28.000000000 +0200
@@ -12,7 +12,7 @@
 	$(CC) -Wall -W $(OBJS) $(LDFLAGS) -o acpitail
 
 install: acpitail
-	cp acpitail /usr/local/sbin
+	cp acpitail ${DESTDIR}/${prefix}
 
 clean:
 	rm -f $(OBJS) acpitail core *.da *.gcov *.bb*
